indico/modules/users/blueprint.py

```diff
@@ -59,6 +60,8 @@
                      'profile_picture_preview', RHProfilePicturePreview)
     _bp.add_url_rule('/preferences/', 'user_preferences', RHUserPreferences, methods=('GET', 'POST'))
     _bp.add_url_rule('/api/preferences/markdown', 'user_preferences_markdown_api', RHUserPreferencesMarkdownAPI)
+    _bp.add_url_rule('/api/preferences/mastodon_server', 'user_preferences_mastodon_server',
```

Comment:
```suggestion
    _bp.add_url_rule('/api/preferences/mastodon-server', 'user_preferences_mastodon_server',
```

underscores in urls are uuuuuuuugly ;)